src/routes/chat/components/messages/Messages.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import map from 'lodash/map';
+import Message from '../message';
+
+function Messages(itemstest) {
```

Comment:
is it a React component? if yes, it's more reasonable to call `props` `props` instead of `itemstest`

Reply:
I alredy change it, and now using data from server not from json file. 
function Messages({ items }) {